app/src/components/AddProposalPanel.js

```diff
@@ -14,23 +14,45 @@ const AddProposalPanel = ({ onSubmit }) => {
   }
   return (
     <form onSubmit={onFormSubmit}>
-      <Field label="Title">
+      <Field
+        label="Title"
+        css={`
+          margin: 16px 0;
```

Comment:
It's better to put a margin on the form tag as it is done in the other apps. See for example:

https://github.com/aragon/aragon-apps/blob/c685c43a99750cb995faaa21754a23b7c7a6cd7c/apps/token-manager/app/src/components/Panels/TokenPanelContent.js#L159-L162